network/src/message/message.rs

```diff
@@ -14,7 +14,6 @@
 // You should have received a copy of the GNU General Public License
 // along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
 
-use snarkos_storage::BlockHeight;
```

Comment:
Why remove this? It's a type alias making it more clear what it refers to; it would be even better to import the related `const` from `snarkvm`, if there is one.

Reply:
iirc it's just a type alias. a wrapper type would actually be nice (like `AleoAmount`) lets make this a new PR though

Reply:
Fair, a strong type would give us some extra safety - yeah, this can be a further PR.